### PR TITLE
add endpoint config and delete all

### LINKED ISSUE #18: Support binance test API
Would be good if we could set the binance URL so we could use the testnet api:

testnet.binance.vision

### PR DESCRIPTION
**Pull request by ghost**
closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ end
 
 Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/bitmex](https://hexdocs.pm/bitmex).
+be found at [https://hexdocs.pm/binance](https://hexdocs.pm/binance).
```
Comment (rupurt) [inline README.md:23]: 👍 😄

File: lib/ex_binance/rest/http_client.ex [hunk 2]
```diff
@@ -109,4 +108,7 @@ defmodule ExBinance.Rest.HTTPClient do
   defp parse_response_body({:ok, %{"code" => _} = reason}), do: {:error, {:binance_error, reason}}
   defp parse_response_body({:ok, _} = result), do: result
   defp parse_response_body({:error, err}), do: {:error, {:decode_error, err}}
+
+  def endpoint, do: "https://#{domain()}"
```
Comment (rupurt) [inline lib/ex_binance/rest/http_client.ex:112]: 👍

File: lib/ex_binance/rest/cancel_all_orders.ex [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  @type symbol :: String.t()
+  @type credentials :: ExBinance.Credentials.t()
+  @type ok_response :: ExBinance.Responses.CancelOrder.t()
+  @type error_msg :: String.t()
+  @type error_reason :: {:not_found, error_msg} | ExBinance.Rest.HTTPClient.shared_errors()
+
+  @path "/api/v3/openOrders"
+  @receiving_window 1000
+
+  @spec cancel_all_orders(symbol, credentials) ::
```
Comment (rupurt) [inline lib/ex_binance/rest/cancel_all_orders.ex:14]: Can you add a test for this? e.g. https://github.com/fremantle-capital/ex_binance/blob/master/test/ex_binance/private/cancel_order_test.exs

**Review by rupurt — CHANGES_REQUESTED** [review]

File: lib/ex_binance/rest/cancel_all_orders.ex (hunk 3 above)
Comment (ghost) [reply lib/ex_binance/rest/cancel_all_orders.ex:14]: Will do.

Comment (ghost) [reply lib/ex_binance/rest/cancel_all_orders.ex:14]: https://github.com/fremantle-capital/ex_binance/blob/dfb0c75385a7df6316f4788a766e147d495ae00a/test/ex_binance/private/cancel_all_orders_test.exs

**Review by rupurt — APPROVED** [review]
Magic 🧙

**Pull request merged**